streamz/core.py

```diff
@@ -1035,6 +1035,15 @@ class unique(Stream):
     parameter.  For example setting ``history=1`` avoids sending through
     elements when one is repeated right after the other.
 
+    Parameters
+    ----------
+    history : int or None, optional
+        number of stored unique values to check against
+    key : function, optional
```

Comment:
You could call this hashfunc to make it clear: it is not a key (in the example, this would be simply "a"), and the purpose is to make the data hashable.